mods/default/furnace.lua

```diff
@@ -232,7 +232,7 @@ local function furnace_node_timer(pos, elapsed)
 	if fuel_totaltime ~= 0 then
 		active = true
 		local fuel_percent = math.floor(fuel_time / fuel_totaltime * 100)
-		fuel_state = S("@1%", fuel_percent)
+		fuel_state = S("@1%", 100-fuel_percent)
```

Comment:
I noticed that '100-fuel percent' is used in the formspec:
```
function default.get_furnace_active_formspec(fuel_percent, item_percent)
	return "size[8,8.5]"..
		"list[context;src;2.75,0.5;1,1;]"..
		"list[context;fuel;2.75,2.5;1,1;]"..
		"image[2.75,1.5;1,1;default_furnace_fire_bg.png^[lowpart:"..
		(100-fuel_percent)..":default_furnace_fire_fg.png]"..
		"image[3.75,1.5;1,1;gui_furnace_arrow_bg.png^[lowpart:"..
```
So i think better to have:
`local fuel_percent = 100 - math.floor(fuel_time / fuel_totaltime * 100)`
As 'fuel percent' logically means 'amount of fuel'.

Reply:
Done!